CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#1854](https://github.com/open-telemetry/opentelemetry-python/pull/1854))
 - Changed AttributeValue sequences to warn mypy users on adding None values to array
   ([#1855](https://github.com/open-telemetry/opentelemetry-python/pull/1855))
+- Added optional `schema_url` field to `Resource` class
+  ([#1862](https://github.com/open-telemetry/opentelemetry-python/issues/1862))
```

Comment:
By the way, this should be the number and link to the PR you submit on the upstream repo. Since you usually wouldn't have submitted it yet, you could just put TBD here until you submit to upstream. Or, you could also use the one you accidentally opened/closed and re-open that PR 😄

Reply:
Done